seismic_zfp/conversion_utils.py

```diff
@@ -59,6 +59,11 @@ def make_header_numpy(bits_per_voxel, blockshape, source, header_info, geom):
                          header_info, bits_per_voxel, blockshape, geom)
     # These 4 bytes indicate the data source for the SGZ file. Use 20 to indicate numpy.
     buffer[76:80] = int_to_bytes(20)
+    if use_higher_samples_precision:
+        # higher precision minimum sample time/depth
```

Comment:
I feel it is more readable if you define a function here

Reply:
this method is only a couple of lines and I think it would actually make it harder to read if we define another function for this

Reply:
I looked at the other parts of the function and it is not needed